src/test/client/syncio.cc

```diff
@@ -77,3 +78,64 @@ TEST_F(TestClient, LlreadvLlwritevInvalidFileHandleSync) {
     rc = client->ll_readv(fh, iov_in, 2, 0);
     ASSERT_EQ(rc, -CEPHFS_EBADF);
 }
+
+TEST_F(TestClient, LlreadvLlwritevLargeBuffersSync) {
```

Comment:
There is a related failure to this test: https://pulpito.ceph.com/vshankar-2025-04-08_07:32:38-fs-wip-vshankar-testing-20250407.173548-debug-testing-default-smithi/8230919/

```
2025-04-09T07:05:46.694 INFO:tasks.workunit.client.0.smithi059.stdout:unknown file: Failure
2025-04-09T07:05:46.694 INFO:tasks.workunit.client.0.smithi059.stdout:C++ exception with description "std::bad_alloc" thrown in the test body.
2025-04-09T07:05:46.694 INFO:tasks.workunit.client.0.smithi059.stdout:
2025-04-09T07:05:46.697 INFO:tasks.workunit.client.0.smithi059.stderr:2025-04-09T07:05:46.621+0000 7f93fccdc680  2 client.4893 unmount
2025-04-09T07:05:46.697 INFO:tasks.workunit.client.0.smithi059.stderr:2025-04-09T07:05:46.621+0000 7f93fccdc680  2 client.4893 unmounting
```

Reply:
@vshankar PTAL

Reply:
Can you explain how that fixes `std::bad_alloc` exception that's being thrown?

Reply:
For simplification, maybe use `UINT_MAX/2`?